### DISCUSSION TITLE
Icon Request: broken link

### DISCUSSION THREAD

**Discussion by sandorfr** [FortAwesome/Font-Awesome#21106] [Icon Requests]
font awesome 4 used to have `chain-broken`: https://fontawesome.com/v4.7.0/icon/chain-broken

it would be nice to have similar icons in font awesome 5 such as a `link-broken` which would work nicely with: https://fontawesome.com/icons/link?style=solid

I understand the current version offers unlink as an alternative, but it's design hardly makes sense when not paired with the link icon. I'm looking for something much more selft explanatory. It does not match the consensus around what a broken link should look like as a google image search shows it.

**Comment by tagliala** [comment]
Already here: https://fontawesome.com/icons/unlink?style=solid

**Comment by sandorfr** [comment]
No, this does not convey the same meaning at all :(
 
(I would even say it does not convey any meaning to me without the text, it looks like a weird paper clip)

**Comment by tagliala** [comment]
As per https://fontawesome.com/how-to-use/upgrading-from-4#icon-name-changes, `unlink` is the replacement icon for `chain-broken`

Feel free to open a new request asking for a design change

**Comment by sandorfr** [comment]
I'm not asking for a redesign as some people might like this icon in some use cases. `unlink` does not mean `broken link`. 

The best would be that you reopen this one and let people vote for it if they agree?

**Comment by tagliala** [comment]
> The best would be that you reopen this one and let people vote for it if they agree?

Since `unlink` is the replacement icon for `chain-broken`, the icon you are basically asking for, I will definitely reopen this if you edit the first post and you explain why `unlink` does not fit the use case

**Comment by sandorfr** [comment]
Done

**Comment by EJlol** [comment]
I'm having the same problem with the unlink one. The unlink icon is simply not clear enough if it is used without the link icon. I also would like to request the link-broken icon as the icon would make more sense  in my application.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Added to 7.2: https://fontawesome.com/icons/link-broken?f=classic&s=solid
